recipes/aws-sdk-cpp/all/conanfile.py

```diff
@@ -336,15 +336,21 @@ def layout(self):
 
     def requirements(self):
         if self.version == "1.11.352":
+            self.requires("aws-crt-cpp/0.26.9", transitive_headers=True)
+            self.requires("aws-c-auth/0.7.16")
+            self.requires("aws-c-cal/0.6.14")
             self.requires("aws-c-common/0.9.15")
+            # self.requires("aws-c-compression/0.2.18")
             self.requires("aws-c-event-stream/0.4.2")
-            self.requires("aws-checksums/0.1.18")
-            self.requires("aws-c-cal/0.6.14")
             self.requires("aws-c-http/0.8.1")
             self.requires("aws-c-io/0.14.7")
-            self.requires("aws-crt-cpp/0.26.9", transitive_headers=True)
+            self.requires("aws-c-mqtt/0.10.3")
             if self.options.get_safe("s3-crt"):
                 self.requires("aws-c-s3/0.5.5")
+            # self.requires("aws-c-sdkutils/0.1.15")
+            self.requires("aws-checksums/0.1.18")
+            # missing aws-lc
+            # self.requires("s2n/1.4.16")
```

Comment:
why not enable aws-c-sdkutils? I see it is listed here https://github.com/aws/aws-sdk-cpp/blob/1b9f1f48db4bb34967508afcbb91b1a1be992b3c/prefetch_crt_dependency.sh#L17

Also s2n https://github.com/aws/aws-sdk-cpp/blob/1b9f1f48db4bb34967508afcbb91b1a1be992b3c/prefetch_crt_dependency.sh#L20